### PR TITLE
docs: fix env-vars.md

### PR DESCRIPTION
**Pull request by jmgiaever**
Add missing env. vars and rename `host` and `port` variables.

### CODE REVIEW

File: docs/guide/env-vars.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 This is the list of the actually supported env vars:
 
 - `NETWORK_KEY`: Zwave Network key
-- `PORT`: The port to listen to for incoming requests. Default is `8091`
```
Comment (robertsLando) [inline docs/guide/env-vars.md:6]: That's not correct, this are used in the backend

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: docs/guide/env-vars.md (hunk 1 above)
Comment (jmgiaever) [reply docs/guide/env-vars.md:6]: Ah, sorry, I though I did a search for it in the code, but guess I missed that. Maybe you can explain what's the difference between SERVER_HOST/-PORT and just the HOST/PORT options in the text.

File: docs/guide/env-vars.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
...
-- `HOST`: The host address to bind to. Default is `0.0.0.0`
 - `STORE_DIR`: The absolute path to the directory where all files will be stored. Default is `<path to your z2m dir>/store`
+- `SERVER_PORT`: The port to listen to for incoming requests. Default is `8091`
+- `SERVER_SSL`: Set to true to use HTTPS / WSS scheme
+- `SERVER_HOST`: The host address to bind to. Default is `0.0.0.0
```
Comment (robertsLando) [inline docs/guide/env-vars.md:9]:
```suggestion
- `SERVER_HOST`: The host address to bind to. Default is `0.0.0.0`
```

File: docs/guide/env-vars.md (hunk 1 above)
Comment (robertsLando) [reply docs/guide/env-vars.md:6]: `SERVER_*` env vars are only used when building using webpack, `PORT/HOST/STORE_DIR` are used from backend when running the application. Could you fix docs according to this please?

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: docs/guide/env-vars.md (hunk 1 above)
Comment (jmgiaever) [reply docs/guide/env-vars.md:6]: Hm, ok. Not sure if I follow. If they're only used for development, they probably shouldn't be here? Variables in this file should probably only list those that is modifiable on startup, and the others should be explained in the dev docs?

Comment (robertsLando) [reply docs/guide/env-vars.md:6]: It's for the frontend, when releasing z2m webpack builds the frontend files in dist folders, those env vars are used in this process, I think them should be documented but not along with the others like it's now, you should split them in another list and specify that are only used in webpack build process

**Comment by robertsLando** [comment]
I have documented them in my other pr #535

**Pull request closed without merging**